content/tutorials/esc-setup-psp-creds/index.md

```diff
@@ -0,0 +1,128 @@
+
+## Create a new ESC project and environment
+
+In ESC, configuration settings belong to collections called [_environments_](/docs/esc/environments/working-with-environments/), and environments in turn belong to _projects_. Projects and environments may be created either with the Pulumi Cloud console or with the CLI.
+
+To get started, navigate to the Environments page using Pulumi Cloud console's navigation bar on the left. Then click `Create Environment` button, fill in `creds` for project name and `psp` for environment name and click `Create`. You will now see a blank new environment with some default commented-out explanations.
+
+## Fill in the PAT
+
+Select everything in the Environment definition pane and replace it with this:
+
+```bash
+values:
+  pulumiConfig:
+    "pulumiservice:apiUrl": % FILL IN CUSTOM BACKEND API HERE %
+    "pulumiservice:accessToken":
+      fn::secret: % FILL IN PAT HERE %
+```
+
+If you are *not* using Pulumi Self-hosted, just remove the line containing `apiUrl`. This optional field configures custom Pulumi Cloud backend, and is not needed for most users.
+
+Once you fill in your previously created PAT, click `Save` on the top right. This will update your new environment's definition with the values you just added. Note how the plain text PAT value you pasted in was converted into a secret. It is now securely stored within Pulumi Cloud, removing the risk of losing or leaking the PAT, while your stacks are still able to decrypt and use it. Feel free to close the token creation page now, we won't need it again!
+
+## Create a new Pulumi Program
+
+Now it's time to create a new Pulumi Program that will use the new Environment and PSP to create more Pulumi Cloud resources!
+
+Create a new folder in your shell (for example `esc-psp-tutorial`) and navigate into it. Then, run `pulumi new`, and navigate the menu to create a new stack from a template called `typescript`. Fill in the project name as `esc-psp-tutorial`, and the stack name as `%ORG NAME%/dev`, filling in your actual organization name (for the rest of the tutorial, `tutorials` will be used), and `dev` is the name of the new stack.
+
+Once Pulumi CLI finishes installing dependencies, you should see the following files in your tutorial folder:
+
+```
+Pulumi.yaml  index.ts  node_modules  package-lock.json  package.json  tsconfig.json
+```
+
+## Import the ESC Environment into the IaC stack
+
+In your tutorial folder, create a new file `Pulumi.dev.yaml`. This file will contain configuration specific to your new `dev` stack. Fill the file in with this:
+
+```
+environment:
+  - creds/psp
+```
+
+In the config files, `environment` field lets you specify an array of ESC Environments to import. In this case, we only want to import the Environment we created above, `creds/psp`, where `creds` is the ESC project name and `psp` is the Environment name.
```

Comment:
"Here, the `environment` field"

probably don't need "where `creds` is the ESC project name and `psp` is the Environment name."